packages/jest-cli/src/lib/__tests__/is_valid_path.test.js

```diff
@@ -61,7 +62,7 @@ it('is not valid when it is a snapshot file', () => {
 it('is not valid when it is a file in the coverage dir', () => {
   expect(
     isValidPath(
-      {},
+      normalize({rootDir}, {}).options,
       config,
       path.resolve(rootDir, 'coverage', 'lib', 'index.js'),
     ),
```

Comment:
Not sure if we still need this test. The same thing is tested in `normalize.test.js`